.github/workflows/check-file.yml

```diff
@@ -26,3 +26,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - run: npm run check:format
+
+  check-branch-is-not-main:
+    name: PR-BRANCH-IS-NOT-MAIN check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jaegertracing/jaeger/.github/actions/block-pr-from-main-branch@main
```

Comment:
please make a copy, these sub-actions are not a public API with guaranteed stability.